### ISSUE TITLE
Please steal the Network Activity Toggle from Core

### ISSUE THREAD

**Issue by Azulan** [gandrewstone/bitcoin#311]
That's the biggest gui/network feature I've been asking for for years.

**Comment by kyuupichan** [comment]
If you want it stolen can you be a lot more explicit about what you're referring to?  For example, the lines of code and/or the PRs in Core that provided this feature?

**Comment by Azulan** [comment]
https://github.com/bitcoin/bitcoin/commit/ab914a65301bd48f2b6c7a64bf57295ae74f2902

**Comment by sickpig** [comment]
@Azulan such PR depends on a lot of other codes merged in core, e.g. the Connection Manager, #8085. So even if the PR and commit you pointed a pretty straight forward all the needed work to get there it's not as quick to merge.

**Comment by GregoryMaxwellIsALiar** [comment]
@Azulan How does this feature improve your user experience? This information helps the team to evaluate the importance of the feature request and to prioritize feature requests.

**Comment by ftrader** [comment]
We don't have anyone called 'GregoryMaxwellIsALiar' on the BU team, and he certainly doesn't speak for us as BU developers.
https://en.wikipedia.org/wiki/False_flag

@Azulan : we look at all feature requests, as you can imagine currently focus is on critical issues of hardening the released code. I think the potential use is clear, but if you want to expand on the benefits, go ahead.

**Comment by GregoryMaxwellIsALiar** [comment]
@ftrader Is it forbidden to help? Do i need a license to make a useful comment? I have years of professional software development experience and i want to help out, why do you behave as i need your permission and why do you behave like a moderator here? Is that your understanding of freedom of speech and open source development?

**All i realized is that the feature request lacks a clear description of the value for the user and if i would realize that this feature is indeed useful i might implement it, that's the way free open source development works, everyone can collaborate, no matter if you like my name or not, it is about the code i write**

**Comment by ftrader** [comment]
I hope you have read the code of the feature on Core. Not sure why you ask the user here to explain the benefit when you can read it yourself - it is a very simple feature.
Either you see the benefit, or you don't - no?

This is different in the case of novel features where a user might be asked to elaborate on what benefits it offers.

**Comment by GregoryMaxwellIsALiar** [comment]
@ftrader No, it is just part of a feature request to explicitly value the benefit of the feature requested... How can i evaluate this feature request, if i was not the one waiting for that feature for years to come??? Why do you want to lead the discussion here instead of allowing free speech?

**Comment by GregoryMaxwellIsALiar** [comment]
@ftrader I don't even see the point why you referenced the short-id collision issue... how is that keeping the discussion on a technical level?
